api/api/clubs/tests.py

```diff
@@ -0,0 +1,22 @@
+from rest_framework.test import APITestCase
+
+from api.clubs.models import Club
+
+
+class ListClubViewTest(APITestCase):
+    """Tests ListClubView for getting the list of clubs."""
+
+    def test_listclubview_no_club_exist_return_empty_list(self):
+        url = '/clubs/'
+        response = self.client.get(url, format='json')
+        self.assertEqual(Club.objects.count(), len(response.data))
+
+    def test_listclubview_countOfClubsinDatabase_equals_countOfClubs_returned(self):
```

Comment:
modify similar to above suggestion.

Reply:
This works `testListClubs_countOfClubsInDb_returnsEqualCountOfClubs`  ?

Reply:
`testListClubs_oneClubExistsInDb_returnsOneClubInResponse`

Make what the testcase checks for painfully clear to the reader.